content/en/seachat/seachat-manual/03-add-knowledge/file-size-rule.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "File Size Rule"
+description: ""
+date: 2024-08-06T08:48:57+00:00
+lastmod: 2024-08-06T08:48:57+00:00
+draft: false
+images: []
+aliases:
+   - /en/seachat/file-size-rule/
+weight: 160 
+toc: true
+---
+
+According to your subscription plan, you can upload files up to a certain size. If you exceed the file size limit, you will receive an error message.
+
+To understand the specific features in different plans, please refer to the [Pricing and Plans](https://wiki.seasalt.ai/seachat/seachat-payments/pricing-plans/).
+
+The file size rule for each plan are as follows:
+
+- **Free** - Knowledge base includes up to 10 documents (200KB max) and 200k tokens
+- **Standard** - Knowledge base includes up to 100 documents (1 MB max) and 1 million tokens
+- **Premium** - Knowledge base includes up to 500 documents (10 MB max) and 5 million tokens
```

Comment:
can you add details on how to check the # of tokens of a document

Reply:
add conclusion from https://github.com/seasalt-ai/ngChat/issues/7862

basically:
for 4o **models**, each row is 8K tokens. for 3.5 model, each spreadsheet row is 2K tokens.